packer/scripts/fedora/ganeti.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash -eux
+
+if [ -e /usr/bin/dnf ] ; then
```

Comment:
@jordane Is there a reason to check execution over existence? All the others are configured to just check existence.